constellations/orion/src/repos/family_repo.go

```diff
@@ -0,0 +1,198 @@
+package repos
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/domains"
+)
+
+// Global variable
+var FamilyRepo FamilyRepoInterface = &familyRepo{}
+
+// Implements interface userRepoInterface
+type familyRepo struct {
+	db *sql.DB // golang native db connection
+}
+
+// Interface to implement
+type FamilyRepoInterface interface {
+	Initialize(db *sql.DB)
+	SelectAll(string, int, int) ([]domains.Family, error)
```

Comment:
I think for now, get rid of SelectAll() for Family.
Some tests will need to change because you're not selecting all families anymore